sdk/storage/azure-storage-common/src/main/java/com/azure/storage/common/implementation/Constants.java

```diff
@@ -88,7 +88,7 @@
     public static final String PROPERTY_AZURE_STORAGE_SAS_SERVICE_VERSION = "AZURE_STORAGE_SAS_SERVICE_VERSION";
 
     public static final String SAS_SERVICE_VERSION = Configuration.getGlobalConfiguration()
-        .get(PROPERTY_AZURE_STORAGE_SAS_SERVICE_VERSION, "2020-10-02");
+        .get(PROPERTY_AZURE_STORAGE_SAS_SERVICE_VERSION, "2020-12-06");
```

Comment:
Is 2020-12-06 supported in canada central already?
If yes, then can we kick off live tests from this PR to confirm?
If no, then we should update https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/storage/azure-storage-common/ci.system.properties instead and keep this one as is until we're ready.

Reply:
According to Sean, the rollout started today. I can switch to updating the properties file instead. Or since the rollout just started and we're only merging into a branch for the apiview, I can leave as is. Either is fine with me.

Reply:
why don't we go directly to main ?

Reply:
btw. we don't have to merge anything in order to generate API view. I'd rather avoid chunking such a small feature into multiple PRs.

Reply:
The goal is to just get the apiview out by the end of this week, which will be more likely if we don't have to worry about writing tests, enabling live tests, etc. We don't have to merge anything into main for an apiview, but we need one location that has all 79 features to get one apiview. That's my goal with this stg79ApiView branch

Reply:
can we merge all the feature branches into stg79ApiView but keep PRs open and eventually target them on main ?

Reply:
Yea I think that's a good idea